specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -4366,13 +4366,24 @@
       "description": "settings for security policy patterns to match",
       "type": "object",
       "properties": {
+        "IsProfileLevel": {
+          "description": "Flag to signal whether the WAF policy association is at the default Profile level.",
+          "type":"boolean"
+        },
         "domains": {
           "description": "List of domains.",
           "type": "array",
           "items": {
             "$ref": "#/definitions/ActivatedResourceReference"
           }
         },
+        "routes": {
+          "description": "List of routes.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ActivatedResourceReference"
```

Comment:
Might need to confirm this should be an 'Activated' reference instead of a plain resource reference.

Reply:
I confirmed with Yunjia (RP team) that routes don't not need to be enabled before linking other resources. If we don't need cx to pass in the route properties, then `ResourceReference` should be sufficient.